docs/index.md

```diff
@@ -650,13 +652,15 @@ Artifactory access tokens may be used via the Authorization header by providing
 block. Getting this value from the environment is supported with the `XRAY_ACCESS_TOKEN`,
 or `JFROG_ACCESS_TOKEN` variables.
 Set `url` field to provide JFrog Xray URL. Alternatively you can set `ARTIFACTORY_URL`, `JFROG_URL` or `PROJECTS_URL` variables.
+Set `skip_version_check` field to true to skip version check. Also you can set `SKIP_XRAY_VERSION_CHECK` environment variable to true to skip version check.
```

Comment:
Set the skip_version_check field to true to skip the version check. Alternatively, you can set the SKIP_XRAY_VERSION_CHECK environment variable to true to skip the version check